src/main/java/com/iota/iri/utils/IotaUtils.java

```diff
@@ -101,4 +109,61 @@ public static ExecutorService createNamedSingleThreadExecutor(String name) {
         return Executors.newSingleThreadExecutor(r -> new Thread(r, name));
 
     }
+	
+	/**
+     * Parses a human readable string for a file size (tb, gb, mb -> file modifier)
+     * Follows the following format: [double][optional space][case insensitive file modifier]
+     * 
+     * Kb is not parsed as this is too small for us to be used in a persistence provider
+     * 
+     * @return The size of the human readable string in bytes, rounded down.
+     */
+    public static long parseFileSize(String humanReadableSize) {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [The method 'parseFileSize(String)' has a cyclomatic complexity of 13.](https://app.codacy.com/manual/GalRogozinski/iri/pullRequest?prid=4453821)